homeassistant/components/sonos/quality_scale.yaml

```diff
@@ -0,0 +1,73 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: |
+      Needs complete refactor
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup:
+    status: exempt
+    comment: |
+      Offline devices are automatically reconnected
+  unique-config-entry:
+    status: exempt
+    comment: |
+      Only allows a single entry
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: todo
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      No authentication
+  test-coverage: todo
```

Comment:
`test_async_poll_manual_hosts_5` why do we set the total second to 0.5 to only sleep 0.5 seconds. Can't we then just sleep the original time?

Reply:
#147064